RocketChat/Rocket.Chat
# regression: Opening DM through UserCard takes too long.

## Proposed changes (including videos or screenshots)





## Issue(s)

[TC-694](https://rocketchat.atlassian.net/browse/TC-694)
## Steps to test or reproduce

1. Open a channel
2. Click a username
3. Click the DM balloon

It should open the DM
## Further comments



[TC-694]: https://rocketchat.atlassian.net/browse/TC-694?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
